index.html

```diff
@@ -5132,6 +5132,10 @@ <h2>The <code>application/credential+ld+json</code> Media Type</h2>
     the use of a specific media type.
   </p>
 
+  <p>
+    This media type can be used with credentials secured using an <a>external proof</a>.
+  </p>
+  <p>If a [[JSON-LD]] context is present in the body of the document, and indicated by the presence of `ld+json` in the media type, the credential MUST be a valid <a href="https://www.w3.org/TR/json-ld11/#dfn-json-ld-document">JSON-LD document</a>.</p>
```

Comment:
Given recent resolutions, I don't think we need this statement at all:

```suggestion
```

We could alternatively change it to:

```suggestion
  <p>A [[JSON-LD]] context is expected to be present in the body of the document, and as indicated by the presence of `ld+json` in the media type, the credential MUST be a valid <a href="https://www.w3.org/TR/json-ld11/#dfn-json-ld-document">JSON-LD document</a>.</p>
```

Reply:
@dlongley your changes seem to make `context` optional, since they change the text from `required` to `expected`.

Is this your intention?

Reply:
> your changes seem to make `context` optional, since they change the text from `required` to `expected`.

No, it doesn't make it optional, it just acknowledges that you cannot expect someone who sends you data tagged with media type `application/anything` to have complied with the requirements of that media type. The fact that it's expected is because it's required. It's a more terse way of writing "Since it's required, parsers will expect it, so make sure you include it as a sender. Of course, as a parser, you should be aware that not every sender will comply, so be ready to reject bad data".

Reply:
I would prefer something to the effect of:

"A context MUST be present in any VCDM media types ending in `+ld+json`."

Reply:
@OR13,

I'm fine with that `MUST` language in a normative section of the spec. That the document is expected to be a valid JSON-LD document comes with the `+ld+json` suffix, but if we don't get that for free (yet), it's also fine to say (or we could use MUST language there as well). People will have opinions around what tests would need to come with this. I think, at this point, we're meaning the same things and we just need everyone to agree on the right way and place to say it.

Reply:
I am going to merge your suggestion after a while if there are no objections.

Reply:
The suggestion we are threaded on: https://github.com/w3c/vc-data-model/pull/1033#discussion_r1123362533

Reply:
I'm trying to process this PR, at present, I don't know how long "after a while if there are no objections" is.

I also don't know if @dlongley's concrete language is the change proposal above, or if it could be considered a part of #1055, or something else.

Help me understand 1) what the final language for this PR is going to be, and 2) when we can merge it.

Reply:
I was waiting to press the "accept suggestion" button, which I just pressed: https://github.com/w3c/vc-data-model/pull/1033#discussion_r1123362533

It automatically marks the thread as resolved.